src/app/CommandHandler.h

```diff
@@ -319,6 +299,54 @@ class CommandHandler
      */
     FabricIndex GetAccessingFabricIndex() const;
 
+    /**
+     * @brief Best effort to add InvokeResponse to InvokeResponseMessage.
+     *
+     * Tries to add response using lambda. Upon failure to add response, attempts
+     * to rollback the InvokeResponseMessage to a known good state. If failure is due
+     * to insufficient space in the current InvokeResponseMessage:
+     *  - Finalizes the current InvokeResponseMessage.
+     *  - Allocates a new InvokeResponseMessage.
+     *  - Reattempts to add the InvokeResponse to the new InvokeResponseMessage.
+     *
+     * addResponseFunction: A lambda function responsible for adding the response
```

Comment:
Please use `@param[in]` syntax.

Reply:
https://github.com/project-chip/connectedhomeip/actions/runs/7576265410/job/20634674678

```
/Users/runner/work/connectedhomeip/connectedhomeip/src/app/CommandHandler.h:312:15: error: parameter 'addResponseFunction:' not found in the function declaration [-Werror,-Wdocumentation]
     * @param addResponseFunction: A lambda function responsible for adding the
              ^~~~~~~~~~~~~~~~~~~~
/Users/runner/work/connectedhomeip/connectedhomeip/src/app/CommandHandler.h:312:15: note: did you mean 'addResponseFunction'?
     * @param addResponseFunction: A lambda function responsible for adding the
              ^~~~~~~~~~~~~~~~~~~~
              addResponseFunction
1 error generated.

Reply:
I think I see what is wrong now. I will address this and see if it fixes the issue